sdk/mobile-center-push/build.gradle

```diff
@@ -0,0 +1,10 @@
+//
```

Comment:
@jaelim-ms I used other modules as example. In analytics and crashes it is same.  So, why here must be different?